src/pybind/mgr/dashboard/frontend/e2e/rgw/userCreate.e2e-spec.ts

```diff
@@ -0,0 +1,228 @@
+      // Checks if we are brought back to the User page after creating
+      // the user
+      page.navigateTo();
+      Helper.getBreadcrumbText().then(function() {});
+      page.getCancel().click();
+      const EC = protractor.ExpectedConditions;
+      const bcrumb = $('.breadcrumb-item.active');
+      browser.wait(EC.visibilityOf(bcrumb), 5000);
+      expect(Helper.getBreadcrumbText()).toEqual('Users');
+    });
+  });
+});
+
+describe('Checks that user was made', () => {
+  let page: UserPage;
+
+  beforeAll(() => {
+    page = new UserPage();
+  });
+
+  describe('Check that the Users username, full name, and email is correct', () => {
+    beforeEach(() => {
+      page.navigateTo();
+    });
+
+    it('should check the username of the user is present', () => {
+      // Makes sure that the users in the table are clickable, then checks if
+      // 000user is in the list
+      page.navigateTo();
+      const EC = protractor.ExpectedConditions;
+      const utable = page.getUserTable();
+      browser.wait(EC.elementToBeClickable(utable.first()), 5000);
+      utable.getText().then(function(userList) {
+        expect(userList).toMatch('000user');
+      });
+    });
+
+    it('should check the full name of the user is present', () => {
+      // Checks if the full name of the user is correct
+      page.navigateTo();
+      page
+        .getUserTable()
+        .getText()
+        .then(function(userList) {
+          expect(userList).toMatch('Sample Name');
+        });
+    });
+
+    it('should check the email of the user is present', () => {
+      // Checks if the email of the user is correct
+      page.navigateTo();
+      page
+        .getUserTable()
+        .getText()
+        .then(function(userList) {
+          expect(userList).toMatch('test@website.com');
+        });
+    });
+  });
+});
+
+describe('Checks that new user was deleted', () => {
+  let page: UserPage;
+
+  beforeAll(() => {
+    page = new UserPage();
+  });
+
+  describe('Selects user, clicks dropdown, selects delete, checks box, and deletes ', () => {
+    beforeEach(() => {
+      page.navigateTo();
+    });
+
+    it('should select the new user and delete it', () => {
+      // Checks if the buttons required to delete the users are present and if
+      // the pop-up menu is present, then clicks the appropriate buttons to delete
+      // the selected user. Also checks if the user has been deleted by comparing the
+      // number of users before and after the delete as well as checking that
+      // 'testuser' is no longer present after the delete
+      page.navigateTo();
+      const EC = protractor.ExpectedConditions;
+      let numDigits = 1;
+      const allu = page.getAllUsers();
+      let count1 = '0';
+      const toggle = page.getToggleMenu();
+      const dtitle = page.getBoxTitle();
+      const confirm = page.getConfirm();
+      const delbutton = page.getDelButton();
+      let count2 = '0';
```

Comment:
The var name "count2" is not very intentional/meaningful, would it be better:
usersAfterDeletion ?